### PR TITLE
Build the feature.

### PR DESCRIPTION
**Pull request by gregglind**

### CODE REVIEW

File: addon/Config.jsm [hunk 1]
```diff
@@ -33,33 +56,15 @@ var config = {
...
     "telemetry": {
       "send": true, // assumed false. Actually send pings?
       "removeTestingFlag": false,  // Marks pings as testing, set true for actual release
       // TODO "onInvalid": "throw"  // invalid packet for schema?  throw||log
     },
     // relative to bootstrap.js in the xpi
-    "studyUtilsPath": `./StudyUtils.jsm`,
+    // "studyUtilsPath": `./StudyUtils.jsm`,
   },
   "isEligible": async function() {
```
Comment (biancadanforth) [inline addon/Config.jsm:69]: I noticed `isEligible` just returns true -- when would we use this method versus using filter expressions in Normandy?

File: addon/bootstrap.js [hunk 2]
```diff
@@ -99,13 +93,17 @@ async function startup(addonData, reason) {
...
+  console.log(new Feature(variation).start());
+  // also set a time to handle "long expiration"
+  // TODO glind
 
 }
 
 
 function shutdown(addonData, reason) {
   console.log("shutdown", studyUtils.REASONS[reason] || reason);
+  const { REASONS } = studyUtils;
```
Comment (biancadanforth) [inline addon/bootstrap.js]: What is this doing here? Are you shortening the path from `studyUtils.REASONS` to just `REASONS`? Kind of like saying: `const REASONS = studyUtils.REASONS`?

File: test/utils.js [hunk 3]
```diff
@@ -25,6 +25,8 @@ const FIREFOX_PREFERENCES = {
   // toolbox.
   "devtools.chrome.enabled": true,
   "devtools.debugger.remote-enabled": true,
+  "devtools.debugger.prompt-connection": false,
+  "general.warnOnAboutConfig": false
```
Comment (biancadanforth) [inline test/utils.js:29]: Great that you added this in. It was on my improvements list. :) I also submitted a [PR](https://github.com/mozilla/shield-studies-addon-utils/pull/92) to set these two prefs in the SSAU repo.

**Review by biancadanforth — COMMENTED** [review]
Mostly questions for you rather than practical feedback! Will look at the telemetry pings separately.

File: addon/bootstrap.js (hunk 2 above)
Comment (gregglind) [reply addon/bootstrap.js]: Yes.  this is called "object destructuring" in es6.

File: test/utils.js (hunk 3 above)
Comment (gregglind) [reply test/utils.js:29]: Good!

**Pull request merged**